### DISCUSSION TITLE
Can I install Ruby On Rails?

### DISCUSSION THREAD

**Discussion by Joaquin694** [leokhoa/laragon#640] [Q&A]
All this week I was looking for a way to install Ruby and Ruby On Rails on my Windows but errors always occur and since I already use Laragon for PHP I wanted to know if it was also possible with Rails?

**Comment by teratai3** [comment]
I am also working on this issue.
I thought it might be an error because laragon ruby does not come with a devkit, but I have not been able to solve it...

**Reply by Joaquin694** [reply]
I managed to insert Ruby into Laragon without problems but I didn't have the DevKit so when I wanted to install Rails it told me that I needed to install MSYS2 but when I installed it on my windows it only gave me an error
